app/views/media_objects/_mejs4_player_js.html.erb

```diff
@@ -16,4 +16,5 @@ var mejs4AvalonPlayer = new MEJSPlayer({
   features: mejs4ConfigFeatures,
   highlightRail: true
 });
+	currentPlayer = mejs4AvalonPlayer.player;
```

Comment:
Same indentation issue in these similar places

Reply:
removed indentations

Reply:
removed indentations